zerver/views/streams.py

```diff
@@ -395,10 +391,6 @@ def get_streams_backend(request, user_profile,
                              include_all_active)
     return json_success({"streams": streams})
 
-def get_public_streams_backend(request, user_profile):
-    return get_streams_backend(request, user_profile, include_public=True,
-                               include_subscribed=False, include_all_active=False)
```

Comment:
It's a similar function, but it's not the same function, and unlike the `alert_words` route, it's used elsewhere (e.g. by the API).  So while it's still possible to replace the old /json/get_public_streams route with the /json/streams route, we need to pass slightly different options to get the same data we had before.